app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -76,8 +76,11 @@ def update
     api :POST, "/content_views/:id/publish", N_("Publish a content view")
     param :id, :identifier, :desc => N_("Content view identifier"), :required => true
     param :description, String, :desc => N_("Description for the new published content view version")
+    param :force_metadata_regeneration, :bool, :desc => N_("Force metadata regeneration on the repositories " \
```

Comment:
Shouldn't we also provide this on the content view version promote?  (API, UI & CLI)